### DISCUSSION TITLE
3.0.1 Vue3/Composition & Quasar not working

### DISCUSSION THREAD

**Discussion by btmi** [dm4t2/vue-currency-input#348] [Q&A] [answered]
Updated from 3.0.0 to 3.0.1 and this broke my implementation.  Calls to setValue are updating the useCurrencyInput reference correctly, but the changes are not being reflected in the referenced q-input.   Had to go back to 3.0.0.

**Comment by dm4t2** [comment, marked as answer]
Sorry for the breaking change! Using `v-model` on `<q-input>` in your component should fix the issue. 
See a similar issue for Element Plus: #347

Working example: https://codesandbox.io/s/vue-currency-input-integration-with-quasar-gwn46

**Reply by btmi** [reply]
Thank you Matthias!!
